cdap-ui/app/services/helpers.js

```diff
@@ -96,11 +96,29 @@ angular.module(PKG.name+'.services')
     return obj;
   }
 
+   function formatDate(date) {
+      var hours = date.getHours();
+      var minutes = date.getMinutes();
+      var seconds = date.getSeconds();
+      var ampm = hours >= 12 ? 'PM' : 'AM';
+      hours = hours % 12;
+      hours = hours ? hours : 12;
+      minutes = minutes < 10 ? '0' + minutes : minutes;
+      seconds = seconds < 10 ? '0' + seconds : seconds;
+      // example format: 1:09:11 PM
+      return hours + ':' + minutes + ':' + seconds + ' ' + ampm;
+    }
+
+    function dateToString(e) {
+      var date = new Date(e * 1000);
+      return formatDate(date);
+    }
```

Comment:
Ok, so this functionality was available.
Take a look how I did it:
https://github.com/caskdata/cdap/pull/2184/files#diff-397d9e155b58c0fb9056a2873515f0afR18

For some reason, $scope was available in the render call, but not $filter. So I put it on the $scope object. Any idea how to get around that?